README.md

```diff
@@ -145,6 +147,8 @@ the length of the box side (in oxDNA simulation units) where the system will be
 text file containing a valid DNA sequence (*e.g.* ATCTGA). If not specified, the sequence will be chosen randomly
 * `-p\--print-virt2-nuc`
 print the `virt2nuc` file that can be used by the oxDNA's `origami_utils.py` script to convert between cadnano and oxDNA nucleotide indexes
+* `-o\--print-oxview`
+print a `.oxview` file that can be opened and edited in oxView [.oxview](https://sulcgroup.github.io/oxdna-viewer/). Using this option will allow you to keep additional design information not included in the oxDNA files.
```

Comment:
I know I'm being obnoxious here, but could fix this line so that it's just [oxView](https://sulcgroup.github.io/oxdna-viewer/)?

Reply:
No worries, that was a typo on my part. Fixed now